hummingbird/ml/operator_converters/sklearn/linear.py

```diff
@@ -39,15 +39,23 @@ def convert_sklearn_linear_model(operator, device, extra_config):
     intercepts = operator.raw_operator.intercept_.reshape(1, -1).astype("float32")
 
     multi_class = None
+    loss = None
     if hasattr(operator.raw_operator, "multi_class"):
         if operator.raw_operator.multi_class == "ovr" or operator.raw_operator.solver in ["warn", "liblinear"]:
             multi_class = "ovr"
         elif operator.raw_operator.multi_class == "auto" and len(classes) == 2:
             multi_class = "ovr"
         else:
             multi_class = "multinomial"
+    if hasattr(operator.raw_operator, "loss"):
+        loss = operator.raw_operator.loss
+        assert (
+            loss == "log" or loss == "modified_huber" or loss == "squared_hinge"
+        ), "predict_proba for linear model currently does not support {}. Please fill an issue at https://github.com/microsoft/hummingbird".format(
+            loss
+        )
```

Comment:
Should we add a check here for other such settings that we don't support?

ex : "[The possible options ](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html)are ‘hinge’, ‘log’, ‘modified_huber’, ‘squared_hinge’, ‘perceptron’, or a regression loss: ‘squared_loss’, ‘huber’, ‘epsilon_insensitive’, or ‘squared_epsilon_insensitive’."

Reply:
Workin on this!